openshift/openshift-virtualization.rst

```diff
@@ -369,16 +369,13 @@ The following example permits the A10 GPU device and A10-24Q vGPU device.
 
       ...
       spec:
-        configuration:
-          developerConfiguration:
-            featureGates:
-            - GPU
-            - DisableMDEVConfiguration
-          permittedHostDevices:
-            pciHostDevices:
-            - externalResourceProvider: true
-              pciDeviceSelector: 10DE:2236
-              resourceName: nvidia.com/GA102GL_A10
+        featureGates:
+          disableMdevConfiguration: true
```

Comment:
I have updated the command called out in the prerequisite section. I have also switched to toggling the feature gate directly in the `HyperConverged` spec  instead of applying the field indirectly through an annotation. This aligns with the instructions we currently have documented for KubeVirt: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-operator-kubevirt.html#prerequisites